lib/api/core/entrypoints/embedded/index.js

```diff
@@ -296,20 +305,33 @@ class EmbeddedEntryPoint extends EntryPoint {
       return;
     }
 
-    const protocol = connection.protocol.toUpperCase();
+    // user init: prioritize the already decoded and verified token stored in
+    // the request
+    // If not available, then that may mean that we didn't verify the user yet,
+    // so we have to decode any provided token
+    let user = null;
+
+    if (request.context.token !== null) {
+      if (request.context.token.userId === this.anonymousUserId) {
+        user = '(anonymous)';
+      } else {
+        user = request.context.token.userId;
+      }
+    }
 
     // = apache combined
+    const protocol = connection.protocol.toUpperCase();
     let
       url,
-      verb = 'DO',
-      user = '-';
+      verb = 'DO';
 
-    if (connection && connection.protocol.startsWith('HTTP/')) {
+    if (connection.protocol.startsWith('HTTP/')) {
       verb = extra.method;
       url = extra.url;
 
-      // try to get plain user name, 1st form jwt token if present, then from basic auth
-      if (connection.headers.authorization) {
+      // try to get plain user name, 1st form jwt token if present, then from
+      // basic auth
+      if (user === null && connection.headers.authorization) {
```

Comment:
Is there any case where `logAccess` could be called before the jwt is set ?

Reply:
I guess on a HTTP `OPTIONS` request?